package.json

```diff
@@ -52,5 +52,8 @@
         "jsdoc": "^3.6.2",
         "redux-logger": "^3.0.6",
         "start-server-and-test": "^1.14.0"
+    },
+    "resolutions": {
+        "@dhis2/ui": "^8.4.4"
     }
```

Comment:
In some previous related PRs I already added this resolutions field to the `v38` and `patch/2.38.1` branches, but I thought it wasn't needed on master. However after upgrading I found quite a few references to the `8.4.2` version in the `yarn.lock`, so I decided to add it here too.